core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -10059,4 +10059,35 @@ private void checkLoptOptimizeJoinRule(LoptOptimizeJoinRule rule) {
     sql(sql).withRule(CoreRules.INTERSECT_TO_EXISTS)
         .checkUnchanged();
   }
+
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-6891">
```

Comment:
Please keep this Java comment consistent with the format of other comments.

Reply:
Thanks, Done.

Reply:
@NobiGo Hi, need I to do something else?